web/app/scripts/nominatim/module.js

```diff
@@ -0,0 +1,8 @@
+(function () {
+    'use strict';
+
+    angular.module('driver.nominatim', [
```

Comment:
I see, that's good reasoning. I'm fine keeping it how it is. If we do need to support multiple geocoders at some point, maybe we can then create a geocoder service which calls out to the individual distinct geocoders.